src/workerd/io/actor-cache.c++

```diff
@@ -106,7 +106,7 @@ ActorCache::Entry::~Entry() noexcept(false) {
   }
 }
 
-ActorCache::SharedLru::SharedLru(Options options): options(options) {}
+ActorCache::SharedLru::SharedLru(const Options options): options(options) {}
```

Comment:
While it can make sense to have a const value parameter in a function definition, I don't think it makes sense to do so here.  Two reasons:
1. The fact that you've declared `this->options` as const means that you can only use `this->options` as const.  The `const` in the parameter declaration doesn't do anything for you here.
2. The `const` in the parameter declaration prevents you from moving the value, e.g., `options(kj::mv(options))`.

Reply:
https://github.com/cloudflare/workerd/pull/1914/commits/a8da46b64c7cd1c429527277c1f1236beda426ed

Yeah, this makes sense. Thanks!